src/View/OpenPatientWindow.py

```diff
@@ -294,7 +294,7 @@ def tree_item_changed(self, item, _):
                 self.selected_series_types.update(series_type)
 
         # Check the existence of IMAGE, RTSTRUCT and RTDOSE files
-        if len(list({'CT', 'MRI', 'PET'} & self.selected_series_types)) == 0:
+        if len(list({'CT', 'MR', 'PET'} & self.selected_series_types)) == 0:
```

Comment:
Might be worthwhile to define different lists, including 'three_d_image_types', plan_types, or whatever it is that appears in more than one place.  That way you update the lists in one place.  Not critical nor urgent.
Technically, these are probably the values in the modality element in DICOM... so names of the lists might say modalities rather than types.